sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_v1/schema.yaml

```diff
@@ -29,6 +29,12 @@ fields:
 - mode: NULLABLE
   name: days_interacted_bits
   type: INTEGER
+- mode: NULLABLE
+  name: days_visited_1_uri_normal_mode_bits
+  type: INTEGER
+- mode: NULLABLE
+  name: days_visited_1_uri_private_mode_bits
+  type: INTEGER
```

Comment:
These fields need to be added to the schema of `telemetry_derived.clients_last_seen_joined_v1`